frontend/packages/console-app/src/components/flags/FeatureFlagExtensionHookResolver.tsx

```diff
@@ -10,7 +10,7 @@ const FeatureFlagExtensionHookResolver: React.FC<FeatureFlagExtensionHookResolve
   handler,
   setFeatureFlag,
 }) => {
-  handler(setFeatureFlag);
+  handler && handler(setFeatureFlag);
```

Comment:
Please mark the handler as optional if it is optional. Btw you can also run such a call with `handler?.(setFeatureFlag)`, but I'm also fine with this condition here.